docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,91 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+---
+
+# Select
+
+<p class="description">Select components are used for choosing options from a list.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+MUI Base offers two components that aim to replace the native `<select>` tag: SelectUnstyled and MultiSelectUnstyled.
```

Comment:
Style, and remove the [weasel words](https://www.amanet.org/articles/get-rid-of-those-pesky-weasel-words/).

```suggestion
MUI Base offers two components to replace the native HTML `<select>` tag: `SelectUnstyled` and `MultiSelectUnstyled`.
```